node/resolvers/Routes/index.ts

```diff
@@ -530,13 +530,20 @@ export const Routes = {
       tradeName,
     })
 
+    const phoneNumberFormatted =
+      phoneNumber ||
+      clUser.phoneNumber ||
```

Comment:
The GraphQL query within the Queryes/Users.ts file is requesting the `phone` and the `homePhone` fields, but there's no `phoneNumber` field there. Maybe this should be `cl.phone` instead, to match the field name? Or is there some other code renaming this field?

```suggestion
      clUser.phone ||
```